### DISCUSSION TITLE
Moderated Sessions removed from OSS

### DISCUSSION THREAD

**Discussion by renatoalmeidaoliveira** [gravitational/teleport#34351] [Q&A] [answered]
I've looked at the changelogs and didnt' find in which version this feature was removed from the OSS, did any one knows? and that feature is something that might be restored to OSS ?

**Comment by zmb3** [comment, marked as answer]
I'm not aware of any point in time where moderated sessions were available in OSS.

In either case, there are no plans to make moderated sessions available in OSS.

**Comment by stevenGravy** [comment]
Peer joining is still available in OSS which might be what you mean. That has changed in roles to require configuring peer joining. Earlier role versions allow peer joining if the user has access to the same user id.
